receiver/otelarrowreceiver/config.go

```diff
@@ -54,3 +64,19 @@ func (cfg *ArrowConfig) Validate() error {
 	}
 	return nil
 }
+
+func (cfg *Config) Validate() error {
+	if err := cfg.GRPC.Validate(); err != nil {
+		return err
+	}
+	if err := cfg.Arrow.Validate(); err != nil {
+		return err
+	}
+	if cfg.Admission.RequestLimitMiB == 0 && cfg.Arrow.DeprecatedAdmissionLimitMiB != 0 {
+		cfg.Admission.RequestLimitMiB = cfg.Arrow.DeprecatedAdmissionLimitMiB
+	}
+	if cfg.Admission.WaiterLimit == 0 && cfg.Arrow.DeprecatedWaiterLimit != 0 {
+		cfg.Admission.WaiterLimit = cfg.Arrow.DeprecatedWaiterLimit
+	}
```

Comment:
I don't think `Validate` should modify the configuration, that should be the work of `Unmarshal`. Could we move this to `Unmarshal`?

Reply:
Fixed.